common-npm-packages/artifacts-common/package.json

```diff
@@ -15,7 +15,7 @@
     "@types/fs-extra": "8.0.0",
     "@types/node": "^10.17.0",
     "@types/mocha": "^5.2.6",
-    "azure-devops-node-api": "10.1.2",
+    "azure-devops-node-api": "10.2.2",
```

Comment:
Since there is a minor bump for node-api - could it make sense to bump minor number for common package also?